src/account-settings/delete-account/ConfirmationModal.jsx

```diff
@@ -107,22 +107,26 @@ export class ConfirmationModal extends Component {
               <PrintingInstructions />
             </p>
           </Alert>
-          <ValidationFormGroup
+          <Form.Group
             for={passwordFieldId}
-            invalid={errorType !== null}
-            invalidMessage={intl.formatMessage(invalidMessage)}
+            isInvalid={errorType !== null}
           >
-            <label className="d-block" htmlFor={passwordFieldId}>
+            <Form.Label className="d-block" htmlFor={passwordFieldId}>
               {intl.formatMessage(messages['account.settings.delete.account.modal.enter.password'])}
-            </label>
-            <Input
+            </Form.Label>
+            <Form.Control
               name="password"
               id={passwordFieldId}
               type="password"
               value={password}
               onChange={onChange}
             />
-          </ValidationFormGroup>
+            {errorType !== null && (
+              <Form.Control.Feedback type="invalid" feedback-for={passwordFieldId}>
+                {intl.formatMessage(invalidMessage)}
```

Comment:
With `ValidationFormGroup` the error message [was wrapped in `<strong>`](https://github.com/openedx/paragon/blob/84350aad7ef8d5e9f9756b2682a0621931ef9f75/src/ValidationFormGroup/index.jsx#L77-L87). This is [not the case with `Form.Control.Feedback`](https://github.com/openedx/paragon/blob/release-23.x/src/Form/FormControlFeedback.jsx).

I don't have strong preferences one way or the other, but if we want to minimize visual changes it might be good to add `<strong>` here

```suggestion
                <strong>{intl.formatMessage(invalidMessage)}</strong>
```

@adamstankiewicz @arbrandes thoughts?

Reply:
I think not having `<strong>` when using `From.Control.Feedback` is perfectly reasonable (as it is the default way to display feedback where `type="invalid"`, so I'm going to resolve this conversation.